malasakit-django/pcari/views.py

```diff
@@ -45,10 +45,8 @@ def wrapper(*args, **kwargs):
 def generate_quantitative_question_ratings_matrix():
     """
     Fetches quantitative question ratings in the form of a numpy matrix.
-
```

Comment:
Put back these newlines so we can clearly delineate each section of the docstring (summary, `Args`, `Returns`, etc.)